docs/sql-ref-syntax-qry-select.md

```diff
@@ -154,7 +154,7 @@ SELECT [ hints , ... ] [ ALL | DISTINCT ] { [ named_expression | regex_column_na
      
 * **regex_column_names**
 
-     When `spark.sql.parser.quotedRegexColumnNames` is true, quoted Identifiers (using backticks) in SELECT
+     When `spark.sql.parser.quotedRegexColumnNames` is true, quoted identifiers (using backticks) in `SELECT`
      statement are interpreted as regular expressions and SELECT statement can take regex-based column specification.
```

Comment:
There's another `SELECT` here

Reply:
Updated

Reply:
Also updated the Doc, seem not very obvious.